File: bialydunajec-angular-frontend/projects/bialydunajec-main/src/app/shared/dto/audit.dto.ts
export class AuditDto {
  createdDate: Date;
  createdBy?: AuditorDto;
  lastModifiedDate?: Date;
  lastModifiedBy?: AuditorDto;
}

export class AuditorDto {
  auditorId: string;
  firstName: string;
  lastName: string;
  emailAddress: string;
}

File: backend/bialydunajec-ddd/dto/src/main/kotlin/org/bialydunajec/ddd/dto/sharedkernel/AuditDto.kt
package org.bialydunajec.ddd.dto.sharedkernel

import java.time.ZonedDateTime

data class AuditDto (
        val createdDate: ZonedDateTime,
        val createdBy: AuditorDto?,
        val lastModifiedDate: ZonedDateTime?,
        val lastModifiedBy: AuditorDto?
)
